spec/motions-spec.coffee

```diff
@@ -1735,6 +1735,25 @@ describe "Motions", ->
       keydown(',')
       expect(editor.getCursorScreenPosition()).toEqual [0, 2]
 
+    it "gets state from GlobalVimState", ->
+      keydown('f')
+      normalModeInputKeydown('c')
+      expect(editor.getCursorScreenPosition()).toEqual [0, 2]
+
+      # store the current find for 'c'
+      findFC = vimState.globalVimState.currentFind
```

Comment:
Instead of accessing `vimState.globalVimState.currentFind` in this test, could we create a second text editor instance and run a find command in it?

I think that something like this should work:

```coffee
helpers.getEditorElement (otherEditorElement) ->
  otherEditorElement.getModel().setText("abc")
  keydown('f', element: otherEditorElement)
  keydown('c', element: otherEditorElement)

  keydown(';')
  expect(editor.getCursorScreenPosition()).toEqual [0, 8]
```